server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -991,13 +991,12 @@ private Modal blockDeleteModal(
       deleteBlockForm
           .withId("block-delete-form")
           .with(
-              div(h1("Are you sure you want to delete this screen?")
-                      .withClasses("text-base", "mb-2"))
-                  .withClasses("mx-4"),
+              div(
+                  h1("Are you sure you want to delete this screen?")
+                      .withClasses("text-base", "mb-4")),
```

Comment:
is mb-2 to mb-4 part of the new design?

Reply:
No quite; because we are changing the full modal container, we needed to make adjustments to several clients of the Modal class so that the padding wasn't too much, otherwise they are indented too far to the right/there is too much space below the modal content.

In principle, the same thing was done with the original Modal container; clients had to adjust their margin/padding to make it look good.